packages/dx-react-scheduler/src/plugins/today-button.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { getMessagesFormatter } from '@devexpress/dx-core';
+import {
+  Plugin,
+  Template,
+  TemplatePlaceholder,
+  TemplateConnector,
+  PluginComponents,
+} from '@devexpress/dx-react-core';
+import { TodayButtonProps } from '../types/today-button';
+
+const pluginDependencies = [
+  { name: 'Toolbar' },
+  { name: 'ViewState' },
+];
+
+const defaultMessages = {
+  today: 'today',
+};
+
+class TodayButtonBase extends React.PureComponent<TodayButtonProps> {
+  static components: PluginComponents = {
+    buttonComponent: 'Button',
+  };
+  render() {
+    const {
+      buttonComponent: Button,
+      messages,
+    } = this.props;
+    const getMessage = getMessagesFormatter({ ...defaultMessages, ...messages });
+
+    return (
+      <Plugin
+        name="TodayButton"
+        dependencies={pluginDependencies}
+      >
+        <Template name="toolbarContent">
+          <TemplateConnector>
+            {(getters, {
```

Comment:
`getters` aren't used, better rename it to `_`

Reply:
I suggest using the `getters` name here because it is more clear to read code